src/core_plugins/interpreter/public/functions/table.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { LegacyResponseHandlerProvider } from 'ui/vis/response_handlers/legacy';
+
+// eslint-disable-next-line new-cap
+const responseHandler = LegacyResponseHandlerProvider().handler;
+
+export default () => ({
+  name: 'kibana_table',
```

Comment:
We could rename this to `data_table`.

Reply:
that would make it ambiguous with canvas `datatable`. in the long term we are gonna get rid of kibana_table and just use `datatable` but for now i would leave kibana in the name of any function where we know its some kind of temporary thing and will only work within kibana, and won't really make any sense inside canvas